deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql

```diff
@@ -260,38 +266,50 @@ WITH rows AS (
             call_trace_address AS trace_address,
             NULL::integer AS evt_index
         FROM (
-            select "output_returnAmount", "amount", "srcToken", "_3" as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" 
-            from oneinch_v3."AggregationRouterV3_call_unoswap" 
-            where call_success 
+            select "output_returnAmount", "amount", "srcToken", "_3" as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
+            from oneinch_v3."AggregationRouterV3_call_unoswap"
+            where call_success
                 and call_block_time >= start_ts
                 and call_block_time < end_ts
             union all
-            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" 
-            from oneinch_v3."AggregationRouterV3_call_unoswapWithPermit" 
+            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
+            from oneinch_v3."AggregationRouterV3_call_unoswapWithPermit"
             where call_success
                 and call_block_time >= start_ts
                 and call_block_time < end_ts
             UNION ALL
-            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" 
-            from oneinch_v4."AggregationRouterV4_call_unoswap"  
-            where call_success 
+            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
+            from oneinch_v4."AggregationRouterV4_call_unoswap"
+            where call_success
                 and call_block_time >= start_ts
                 and call_block_time < end_ts
             union all
-            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address" 
-            from oneinch_v4."AggregationRouterV4_call_unoswapWithPermit"  
+            select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
+            from oneinch_v4."AggregationRouterV4_call_unoswapWithPermit"
+            where call_success
+                and call_block_time >= start_ts
+                and call_block_time < end_ts
+            union all
+            select "output_returnAmount", "amount", "srcToken", numeric2bytea(pools) as pools, "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
```

Comment:
Thank you @fatayri. The code fails during runtime on ` numeric2bytea(pools)` with the following error:

> No function matches the given name and argument types. You might need to add explicit type casts.

Reply:
See also https://dune.com/queries/1714974:

> Error: Function numeric2bytea(numeric[]) does not exist at line 5, position 3.